sdk/cosmos/azure-cosmos/azure/cosmos/_session_token_helpers.py

```diff
@@ -0,0 +1,224 @@
+# The MIT License (MIT)
+# Copyright (c) 2014 Microsoft Corporation
+
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Internal Helper functions for manipulating session tokens.
+"""
+from typing import Tuple, List, Dict, Any
+
+from azure.cosmos._routing.routing_range import Range
+from azure.cosmos._vector_session_token import VectorSessionToken
+from ._change_feed.feed_range_internal import FeedRangeInternalEpk
+
+# pylint: disable=protected-access
+
+
+# ex inputs:
+# 1. "1:1#51", "1:1#55" -> "1:1#55"
+# 2. "0:1#57", "1:1#52" -> "0:1#57"
+def merge_session_tokens_with_same_range(session_token1: str, session_token2: str) -> str:
+    pk_range_id1, vector_session_token1 = parse_session_token(session_token1)
+    pk_range_id2, vector_session_token2 = parse_session_token(session_token2)
+    pk_range_id = pk_range_id1
+    # The partition key range id could be different in this scenario
+    # Ex. get_updated_session_token([("AA", "BB"), "1:1#51"], ("AA", "DD")) -> "1:1#51"
```

Comment:
nit: For the ease of understanding, array of tuples of _feed range and session token_ can be specified like this - [(("AA", "BB"), "1:1#51")]

Reply:
Fixed the examples to be in that format.